<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>弹幕频道设置</string>
  </property>
  <widget class="QLabel" name="serverLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="channelLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>频道名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>频道密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>http://vpn.wuqiong.tk:5000/</string>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="channelLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>demo</string>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>202</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>202</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createChannelButton">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>232</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>创建频道</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterChannelButton">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>172</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>进入频道</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>279</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Copyright © 2013-2015  &lt;a href=&quot;http://www.diveinedu.com&quot;&gt;戴维营教育&lt;/a&gt; All Rights Reserved.   </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="channelLabel_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请检查频道是否过期, 否则重新创建: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
